event.go

```diff
@@ -362,7 +362,15 @@ func (e *Event) Pretty() (out string, ok bool) {
 
 			return fmt.Sprintf("[*] CTCP query from %s: %s%s", ctcp.Source.Name, ctcp.Command, " "+ctcp.Text), true
 		}
-		return fmt.Sprintf("[%s] (%s) %s", strings.Join(e.Params, ","), e.Source.Name, e.Trailing), true
+
+		var source string
+		if e.Command == PRIVMSG {
+			source = fmt.Sprintf("(%s)", e.Source.Name)
+		} else { // NOTICE
+			source = fmt.Sprintf("--%s--", e.Source.Name)
```

Comment:
`**%s**` is already used for `/me` so I figured `--%s--` might be a good derivation? If anyone has a better idea let me know :)